demo-microservices/gateway-service/src/main/java/org/springdoc/demo/services/gateway/GatewayApplication.java

```diff
@@ -31,7 +31,8 @@ public List<GroupedOpenApi> apis(RouteDefinitionLocator locator) {
 		}
 		definitions.stream().filter(routeDefinition -> routeDefinition.getId().matches(".*-service")).forEach(routeDefinition -> {
 			String name = routeDefinition.getId().replaceAll("-service", "");
-			GroupedOpenApi.builder().pathsToMatch("/" + name + "/**").group(name).build();
+			GroupedOpenApi api = GroupedOpenApi.builder().pathsToMatch("/" + name + "/**").group(name).build();
```

Comment:
It seems, this Bean definition has no impact. The groups are created based on `springdoc.swagger-ui.urls` config.

https://github.com/springdoc/springdoc-openapi-demos/blob/2.x/demo-microservices/config-service/src/main/resources/config/gateway-service.yml#L66

I commented out this bean definition and it still works.